web/src/documents.ts

```diff
@@ -64,6 +64,21 @@ export interface ContentChangeEvent extends CustomEvent {
   detail: ViewUpdate | string
 }
 
+/**
+ * Create a new document
+ * 
+ * Optionally pass the content, in some format, for the new document.
+ */
+export async function create(
+  client: Client,
+  content?: string,
+  format?: string
+): Promise<Document> {
```

Comment:
I wonder if we should go with the convention of using objects as parameters, make arguments a bit more self documenting, and account for any future API changes.

Reply:
Finally replying to this. I'm OK with using object as parameters as long as it does not introduce too much boilerplate.

We have talked about refactoring towards using classes for `Document`, `Session` etc in which case parameters such as `client` and `documentId` will be moved into class properties. So perhaps lets revisit convention for parameters when / if we do that refactoring.